app/src/main/java/org/simple/clinic/monthlyReports/questionnaire/sync/QuestionnariePayload.kt

```diff
@@ -0,0 +1,26 @@
+package org.simple.clinic.monthlyReports.questionnaire.sync
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import org.simple.clinic.monthlyReports.questionnaire.Questionnaire
+import org.simple.clinic.monthlyReports.questionnaire.QuestionnaireType
+import java.util.UUID
+
+@JsonClass(generateAdapter = true)
+data class QuestionnairePayload(
+    @Json(name = "id")
+    val uuid: UUID,
+
+    @Json(name = "questionnaire_type")
+    val questionnaireType: QuestionnaireType,
+
+    @Json(name = "layout")
+    val layout: String,
```

Comment:
Does the Moshi library automatically parse the JSON object to string or do we need to add a type converter? Please verify that with a local JSON if API isn't ready yet.